controllers/user.js

```diff
@@ -69,15 +69,29 @@ async function updateCommunicationOptions(req, res) {
   return res.json("Comm options updated");
 }
 
-async function add(req, res) {
-  // need to decide how to handle resends
 
-  _requireSessionUser(req);
+function _checkForDuplicateEmail(sessionUser, email) {
+  if (email === sessionUser.primary_email) {
+    throw new FluentError("user-add-duplicate-email");
```

Comment:
Can you confirm this ID? I did a quick <kbd>git grep "user-add-duplicate-email"</kbd> and didn't find any results in **master** branch in any of the .ftl files.

Reply:
I also ran <kbd>git grep "new FluentError" | sort | uniq</kbd> and didn't find any .ftl results for the "must-be-signed-in" key:

```sh
git grep "must-be-signed-in" locales/en/
```

Unrelated, but I also spotted this on L92 below, which is inconsistent with other `FluentError()` usages:

```js
throw new FluentError("Error message for this verification email timed out or something went wrong.");
```